### ISSUE TITLE
Indent line missing in vscode merge editor

### ISSUE THREAD

**Issue by d1313113** [Monokai/monokai-pro-vscode#317]
Indent line missing in vscode merge editor

<img width="1006" alt="image" src="https://user-images.githubusercontent.com/35473748/186853112-c75da5dc-78ec-4666-9553-67bdc91ee0b0.png">

<img width="998" alt="image" src="https://user-images.githubusercontent.com/35473748/186853083-4c1b4f6f-ebab-4188-8c96-85f4e4f09b30.png">

**Comment by d1313113** [comment]
```json
{
  "workbench.colorCustomizations": {
    "checkbox.border": "#ffffff" // set this fixed the problem
  }
}
```

**Comment by Monokai** [comment]
fixed in 1.1.21

**Comment by d1313113** [comment]
thanks,after update the extension, the problem was fixed
